pkg/settings/setting.go

```diff
@@ -538,6 +539,14 @@ func IsReleaseServerVersion(serverVersion string) bool {
 	return true
 }
 
+// IsPlainIPV6 will return true if the given address is a plain IPV6 address and not encapsulated or similar.
+func IsPlainIPV6(address string) bool {
```

Comment:
I'm not sure if `settings` is the correct package for this, but we don't have a `pkg/util` so maybe `settings` is an okay replacement for that.